.github/workflows/release.yml

```diff
@@ -4,23 +4,19 @@ on:
     branches:
       - master
 jobs:
-  prepare:
-    runs-on: ubuntu-latest
-    if: "! contains(github.event.head_commit.message, '[skip ci]')"
-    steps:
-      - run: echo "${{ github.event.head_commit.message }}"
```

Comment:
I'm wondering we you've removed this step? :) I think it's still necessary

Reply:
This is natively supported for some time now: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/

That's why I think it isn't needed any longer :)

Reply:
Thanks for clarification. I didn't know that :)